docs/ManualMocks.md

```diff
@@ -92,7 +104,13 @@ fs.readdirSync = readdirSync;
 module.exports = fs;
 ```
 
-Now we write our test. Note that we need to explicitly tell that we want to mock the `fs` module because it’s a core Node module:
+Now we write our test.
+
+:::note
+
+We need to explicitly tell that we want to mock the `fs` module because it’s a core Node module:
+
+:::
```

Comment:
Colon at the end of a note does not look good. Better not to split the paragraph here.

The second sentence could also become a code comment.

```suggestion
Now we write our test. In this case `jest.mock('fs')` must be called explicitly, because `fs` is Node’s build-in module:
```